src/program/lwaftr/run_nohw/README

```diff
@@ -4,14 +4,15 @@ Usage: run-nohw --help
        --conf, -c PATH      Path to the lwAFTR configuration file.
```

Comment:
(Github won't let me add this comment two lines above, where it should go.)

The main `README` and all example mention `run_nohw` with an underscore, rather than with a dash as shown here.